src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+html {
+  --main-color: #000;
+  --active-color: #00acdc;
+  --background-color: #fff;
+  box-sizing: border-box;
```

Comment:
More correctly to move this rule to `*` selector, because to apply this rule to all elements